src/tbbbind/tbb_bind.cpp

```diff
@@ -405,7 +405,7 @@ class binding_handler {
     affinity_masks_container affinity_backup;
     system_topology::affinity_mask handler_affinity_mask;
 
-#if WIN32
+#ifdef _WIN32
```

Comment:
In other places `#if` is preferred over `#ifdef` (even for `_WIN32`), is it critical to have `#ifdef`?

Reply:
ifdef is best to avoid warnings when its undefined on other platforms but I dont think its critical

Reply:
> ifdef is best to avoid warnings when its undefined on other platforms but I dont think its critical

It was an error not a warning.
If they prefer #if then they should include "windows.h" (or whatever the file that define WIN32=1), I think!

Reply:
> ifdef is best to avoid warnings when its undefined on other platforms but I dont think its critical

Are all the warnings are eliminated with the patch? (it seems there are a lot of `#if something_is_undefined` expressions across the code base)

Reply:
no, you will see that tbb throws a lot of warnings if you enable the gcc warning "-Wundef"
but at least it will build for mingw